README.md

```diff
@@ -2,9 +2,225 @@
 
 [![Build Status](https://travis-ci.org/revdotcom/revai-java-sdk.svg?branch=develop)](https://travis-ci.org/revdotcom/revai-java-sdk)
 
+## Documentation
+
+See the [API docs](https://www.ai.rev.ai/docs) for more information about the API.
+
+## Install the SDK
+The recommended way to use the Rev.ai Java SDK is to import it into the project using Maven.
+
+      <dependency>
+        <groupId>ai.rev</groupId>
+        <artifactId>revai-java-sdk</artifactId>
+        <version>1.0.0</version>
+      </dependency>
+    
+## Installing it locally
+Once you've cloned the repo you can use Maven to build it locally and install it in your local Maven .m2 repository.
+
+    mvn install -Dmaven.test.skip=true
+
+## Support
+
+We support Java 8 and 11.
+
+## Usage
+
+All you need to get started is your Access Token, which can be generated on
+your [Settings Page](https://www.ai.rev.ai/access_token). Create a client with the
+given Access Token:
+
+```
+// Initialize your client with your ai.rev access token
+String accessToken = "Your Access Token";
+ApiClient apiClient = new ApiClient(accessToken);
+```
+
+### Checking credits remaining
+
+```
+RevAiAccount revAiAccount = apiClient.getAccount(); 
+```
+
+### Submitting a job
+
+You can submit a local file
+```
+String localPathToFile = "./path/to/file.mp3";
+RevAiJob revAiJob = apiClient.submitJobLocalFile(localPathToFile);
+```
+
+or submit via a public direct download url
+```
+String urlLinkToFile = "https://www.rev.ai/FTC_Sample_1.mp3";
+RevAiJob revAiJob = apiClient.submitJobUrl(urlLinkToFile);
+```
+
+or from FileInputStream, the filename is optional.
+```
+File file = new File("./path/to/file.mp3");
+FileInputStream fileInputStream;
+try {
+  fileInputStream = new FileInputStream(file);
+} catch (FileNotFoundException e) {
+  throw new RuntimeException("Could not find file [" + file.getName() + "]");
+}
+RevAiJob revAiJob = apiClient.submitJobLocalFile(fileInputStream, String fileName, RevAiJobOptions options);
+```
+
+`RevAiJob` objects contain job information as defined by the [documentation](https://www.ai.rev.ai/docs#operation/SubmitTranscriptionJob).
+
+If you want to get fancy, all submit job methods have overrides that allow specifying 
+[RevAiJobOptions](./src/main/java/ai/rev/models/asynchronous/RevAiJobOptions.java) to configure job specific settings. These are also described in the 
+request body of the [Submit Job](https://www.ai.rev.ai/docs#operation/SubmitTranscriptionJob) 
+endpoint.
+
+### Checking your job's status
+
+You can check the status of your transcription job using its `id`
+
+```
+RevAiJob newlyRefreshedRevAiJob = apiClient.getJobDetails(revAiJob.getJobID());
```

Comment:
small nitpick on the theme of naming, but could we use `JobId` instead of `JobID`everywhere (and in general Id vs ID). Generally we use Id in all of our other code.

Reply:
Good catch!